src/app/search/FilterHelp.tsx

```diff
@@ -0,0 +1,700 @@
+                  <span dangerouslySetInnerHTML={{ __html: t('Filter.Source.EV') }} />
+                  <span dangerouslySetInnerHTML={{ __html: t('Filter.Source.Gunsmith') }} />
+                  <span dangerouslySetInnerHTML={{ __html: t('Filter.Source.Shipwright') }} />
+                  <span dangerouslySetInnerHTML={{ __html: t('Filter.Source.EDZ') }} />
+                  <span dangerouslySetInnerHTML={{ __html: t('Filter.Source.Titan') }} />
+                  <span dangerouslySetInnerHTML={{ __html: t('Filter.Source.Nessus') }} />
+                  <span dangerouslySetInnerHTML={{ __html: t('Filter.Source.Io') }} />
+                  <span dangerouslySetInnerHTML={{ __html: t('Filter.Source.Mercury') }} />
+                  <span dangerouslySetInnerHTML={{ __html: t('Filter.Source.Mars') }} />
+                  <span dangerouslySetInnerHTML={{ __html: t('Filter.Source.Tangled') }} />
+                  <span dangerouslySetInnerHTML={{ __html: t('Filter.Source.Dreaming') }} />
+                  <span dangerouslySetInnerHTML={{ __html: t('Filter.Source.Crucible') }} />
+                  <span dangerouslySetInnerHTML={{ __html: t('Filter.Source.Trials') }} />
+                  <span dangerouslySetInnerHTML={{ __html: t('Filter.Source.IronBanner') }} />
+                  <span dangerouslySetInnerHTML={{ __html: t('Filter.Source.Nightfall') }} />
+                  <span dangerouslySetInnerHTML={{ __html: t('Filter.Source.Zavala') }} />
+                  <span dangerouslySetInnerHTML={{ __html: t('Filter.Source.Ikora') }} />
+                  <span dangerouslySetInnerHTML={{ __html: t('Filter.Source.Gambit') }} />
+                  <span dangerouslySetInnerHTML={{ __html: t('Filter.Source.EP') }} />
+                  <span dangerouslySetInnerHTML={{ __html: t('Filter.Source.Raid') }} />
+                  <span dangerouslySetInnerHTML={{ __html: t('Filter.Source.Leviathan') }} />
+                  <span dangerouslySetInnerHTML={{ __html: t('Filter.Source.SoS') }} />
+                  <span dangerouslySetInnerHTML={{ __html: t('Filter.Source.EoW') }} />
+                  <span dangerouslySetInnerHTML={{ __html: t('Filter.Source.LastWish') }} />
+                  <span dangerouslySetInnerHTML={{ __html: t('Filter.Source.Prestige') }} />
+                  <span dangerouslySetInnerHTML={{ __html: t('Filter.Source.Adventure') }} />
+                </td>
+              </tr>
+            )}
+            {destinyVersion === 1 && (
+              <tr>
+                <td>
+                  <span>is:vanilla</span> <span>is:qw</span> <span>is:ib</span> <span>is:vog</span>
+                  <span>is:ce</span> <span>is:poe</span> <span>is:trials</span> <span>is:ttk</span>
+                  <span>is:coe</span> <span>is:kf</span> <span>is:srl</span> <span>is:cd</span>
+                  <span>is:roi</span> <span>is:af</span> <span>is:wotm</span>{' '}
+                  <span>is:dawning</span>
+                  <span>is:aot</span>
```

Comment:
```
                  <span>is:dawning</span> <span>is:aot</span>
```

Reply:
Also not sure what the `{' '}` are for, they are only on some lines.

Reply:
React by default collapses spaces between lines, so that inserts a space back in.